<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Portal 2: Multiplayer Mod</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/Images/MainWindowIcon.png</normaloff>Resources/Images/MainWindowIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 63 20pt &quot;Quicksand SemiBold&quot;;
color: rgb(104, 150, 137);
background-color: rgb(11, 60, 73);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button_Play">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 231, 69);
font: 600 30pt &quot;Quicksand&quot;;</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Resources/Images/logo.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Public IP:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_CopyIP">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>123.456.789.123</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Images/copy-icon.svg</normaloff>Resources/Images/copy-icon.svg</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#da2c38;&quot;&gt;Developer Mode:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Button_DeveloperMode">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Guide">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>390</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 231, 69);
border-color: rgb(19, 231, 69);
font: 600 30pt &quot;Quicksand&quot;;</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Discord">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>390</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 231, 69);
border-color: rgb(19, 231, 69);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Images/discord-icon.svg</normaloff>Resources/Images/discord-icon.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Unmount">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(218, 44, 56);
font: 600 30pt &quot;Quicksand&quot;;</string>
    </property>
    <property name="text">
     <string>Unmount</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>40</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings_2">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGuide"/>
    <addaction name="actionCheck_for_update"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSettings_2"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuide">
   <property name="text">
    <string>Guide</string>
   </property>
  </action>
  <action name="actionCheck_for_update">
   <property name="text">
    <string>Check for update</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
